frontend/packages/employee-frontend/src/components/reports/VoucherServiceProviderUnit.tsx

```diff
@@ -0,0 +1,249 @@
+                },
+                {
+                  label: i18n.reports.voucherServiceProviderUnit.coefficient,
+                  key: 'serviceVoucherServiceCoefficient'
+                }
+              ]}
+              filename={`Palvelusetelilapset yksikössä ${unitName} ${filters.month}/${filters.year}.csv`}
+            />
+            <TableScrollable>
+              <Thead>
+                <Tr>
+                  <Th>{i18n.reports.voucherServiceProviderUnit.child}</Th>
+                  <Th>{i18n.reports.common.groupName}</Th>
+                  <Th>
+                    {i18n.reports.voucherServiceProviderUnit.numberOfDays}
+                  </Th>
+                  <Th>
+                    {
+                      i18n.reports.voucherServiceProviderUnit
+                        .serviceVoucherValue
+                    }
+                  </Th>
+                  <Th>
+                    {
+                      i18n.reports.voucherServiceProviderUnit
+                        .serviceVoucherCoPayment
+                    }
+                  </Th>
+                  <Th>
+                    {
+                      i18n.reports.voucherServiceProviderUnit
+                        .serviceVoucherRealizedValue
+                    }
+                  </Th>
+                  <Th>{i18n.reports.voucherServiceProviderUnit.coefficient}</Th>
+                  <Th>{i18n.reports.voucherServiceProviderUnit.serviceNeed}</Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                {rows.data.map((row: VoucherServiceProviderUnitRow) => (
+                  <Tr key={`${row.childId}`}>
+                    <Td>
+                      <Link to={`/child-information/${row.childId}`}>
+                        {formatName(
+                          row.childFirstName,
+                          row.childLastName,
+                          i18n
+                        )}
+                      </Link>
+                      <br />
+                      {row.childDateOfBirth.format()}
+                    </Td>
+                    <Td>{row.childGroupName}</Td>
+                    <Td>
+                      <Tooltip
+                        tooltip={
+                          <div>
+                            {row.derivatives.realizedPeriod.start.format()} -{' '}
+                            {row.derivatives.realizedPeriod.end.format()}
+                          </div>
+                        }
+                      >
+                        {row.derivatives.numberOfDays}
+                      </Tooltip>
+                    </Td>
+                    <Td>{formatCents(row.serviceVoucherValue)}</Td>
+                    <Td>{formatCents(row.serviceVoucherCoPayment)}</Td>
+                    <Td>{formatCents(row.derivatives.realizedAmount)}</Td>
+                    <Td>{row.serviceVoucherServiceCoefficient}</Td>
+                    <Td>{formatServiceNeed(row.serviceVoucherHoursPerWeek)}</Td>
```

Comment:
By the way, "sopimuspäivät" is not just service need hours per week. It's a Tampere specific set of special service needs. I would leave it out completely for now.

Reply:
Yes well from the spec I understood that for Tampere this should be something different whereas for Espoo service need hours would suffice. But I really don't know the story behind this :man_shrugging:  Column could be left out of course if that's better.

Reply:
Maybe just renaming the header is enough then?

Reply:
Good thinking sir :+1: